### PR TITLE
add a starter prompt for what to do when handing off

### PR DESCRIPTION
**Pull request by heatherm**

### CODE REVIEW

File: hand_offs.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- [  ] Mapping services?
+- [  ] Mail Service
+- [  ] Pingdom/Alerts
+- [  ] CI/CD stuff
+- [  ] Database URL and credentials?
+- [  ] Analytics accounts
+- [  ] Scheduler services?
+- [  ] SMS/Push services?
+- [  ] Product management backlog (Asana or Tracker?)
+1. Guide PM through secure transfer of .env file (if we are passing on accounts
```
Comment (hcarreras) [inline hand_offs.md]: It would be great to know how to securely transfer secrets. I'm not sure what a mapbox is. Probably not related to https://www.mapbox.com/

File: hand_offs.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+well as any known issues.
+
+Heroku apps
+1. Make sure all related Heroku plugins are pointed to client emails (ex:
+librato alerts or rollbar alerts)
+1. Make sure all PRs are closed and push to heroku, then transfer the pipeline 
+in the GUI. Client will setup their own github with what they pull from Heroku.
+
+
+PHP apps
```
Comment (hcarreras) [inline hand_offs.md]: Why is this related to PHP only?

Comment (heatherm) [reply hand_offs.md]: Good question :) Going thought is that PHP apps don't use Heroku (or if they do, the clients we have do not want Heroku transferred). So, for our current suite of clients with php apps, we should shut down Heroku and transfer Github. For our rails clients, the opposite is true. They all want Heroku, and will create their own githubs.

We could also always handoff both...

File: hand_offs.md (hunk 1 above)
Comment (heatherm) [reply hand_offs.md]: Yeah, 100% is https://www.mapbox.com/. It's an example pulled from Gehl where we signed up for the service for them to get a mapping API key.

Comment (hcarreras) [reply hand_offs.md]: Ah. Now, I understand. I found it a bit confusing, so perhaps other devs will find it confusing too.
```suggestion
1. Guide PM through a secure transfer of .env file (if we are passing on accounts with keys that will be transitioned to clients, we'll need to transfer the private keys for all the services we signed up for on their behalf)
```

Comment (hcarreras) [reply hand_offs.md]: I think we should specify the way of sharing the .env file. 
The most practical/secure I can think of is:
1. Have `.env` encrypted using [Gitcrypt](https://github.com/AGWA/git-crypt). Store encryption key in 1Password
2. When handing over to a client, [invite the client as a guest to 1password](https://support.1password.com/guests-teams/). Grant permissions to see Gitcypt's encryption key only.

Comment (heatherm) [reply hand_offs.md]: What do you think about using the built in rails secrets vs gitcrypt? https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2
